### PR TITLE
feat: generate color utilities

### PR DESCRIPTION
**Pull request by annsch**
- [x] add color utilities for all neutral backgrounds

### CODE REVIEW

**Comment by nmerget** [comment]
source/_patterns/spacings/_examples.demonstration.scss -> 	background-color: $db-colors-primary-bg-enabled;

This should be renamed as well.

File: scripts/color-classes-generator.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        @extend %${prefix}-bg-${value}-ia;
+    }
+
+    a {
+        @extend %${prefix}-bg-${value}-text-ia;
+    }
+}`;
+		}
+
+		// 		// special case neutral has no default value for enabled
```
Comment (nmerget) [inline scripts/color-classes-generator.js]: Do we need this comment?

Comment (annsch) [reply scripts/color-classes-generator.js]: removed it before ..

File: scripts/color-classes-generator.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+			output += `
+.${prefix}-text-${value} {
+    @extend %${prefix}-text-${value};
+        
+    &-ia,
+    &[data-variant="ia"] {
+        @extend %${prefix}-text-${value}-ia;
+    }
+
+    a {
```
[the remark this answers is not in the document]
Comment (annsch) [reply scripts/color-classes-generator.js:63]: yes, of course :)

**Review by nmerget — APPROVED** [review]
LGTM

**Pull request merged**